README.md

```diff
@@ -4,6 +4,13 @@
 
 Provides OpenTelemetry Exporters for Google Cloud Operations. 
 
+To get started with instrumentation in Google Cloud, see [Generate traces and metrics with
+Java](https://cloud.google.com/stackdriver/docs/instrumentation/setup/java).
+
+To learn more about instrumentation and observability, including opinionated recommendations
```

Comment:
I think opinionated recommendations should point to the choose your instrumentation approach?

Reply:
It's linked from the Overview page in the "General recommendations" section, so I was thinking to minimize the amount of duplicated info and not reproduce the ToC